<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigsForm</class>
 <widget class="QDialog" name="ConfigsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>252</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>373</width>
    <height>252</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>252</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Настройки</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="addNewTrackToPlaylist_ChBox">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Добавлять новый трек в сразу в список воспроизведения</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="resizeByCount_checkBox">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Подстраивать вертикальный размер под число аудиодорожек</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="volumeView_GBox">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Вид регулятора громкости</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="dialView_checkBox">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Крутилка</string>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="sliderView_checkBox">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Ползунок</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QDial" name="dialVolume">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="notchTarget">
         <double>10.000000000000000</double>
        </property>
        <property name="notchesVisible">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSlider" name="sliderVolume">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksAbove</enum>
        </property>
        <property name="tickInterval">
         <number>10</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="accept_btn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="Button">
           <brush brushstyle="LinearGradientPattern">
            <gradient startx="0.000000000000000" starty="0.000000000000000" endx="0.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
             <gradientstop position="0.000000000000000">
              <color alpha="255">
               <red>144</red>
               <green>238</green>
               <blue>144</blue>
              </color>
             </gradientstop>
             <gradientstop position="1.000000000000000">
              <color alpha="255">
               <red>0</red>
               <green>128</green>
               <blue>0</blue>
              </color>
             </gradientstop>
            </gradient>
           </brush>
          </colorrole>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>243</red>
             <green>243</green>
             <blue>243</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base">
           <brush brushstyle="LinearGradientPattern">
            <gradient startx="0.000000000000000" starty="0.000000000000000" endx="0.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
             <gradientstop position="0.000000000000000">
              <color alpha="255">
               <red>144</red>
               <green>238</green>
               <blue>144</blue>
              </color>
             </gradientstop>
             <gradientstop position="1.000000000000000">
              <color alpha="255">
               <red>0</red>
               <green>128</green>
               <blue>0</blue>
              </color>
             </gradientstop>
            </gradient>
           </brush>
          </colorrole>
          <colorrole role="Window">
           <brush brushstyle="LinearGradientPattern">
            <gradient startx="0.000000000000000" starty="0.000000000000000" endx="0.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
             <gradientstop position="0.000000000000000">
              <color alpha="255">
               <red>144</red>
               <green>238</green>
               <blue>144</blue>
              </color>
             </gradientstop>
             <gradientstop position="1.000000000000000">
              <color alpha="255">
               <red>0</red>
               <green>128</green>
               <blue>0</blue>
              </color>
             </gradientstop>
            </gradient>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Button">
           <brush brushstyle="LinearGradientPattern">
            <gradient startx="0.000000000000000" starty="0.000000000000000" endx="0.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
             <gradientstop position="0.000000000000000">
              <color alpha="255">
               <red>144</red>
               <green>238</green>
               <blue>144</blue>
              </color>
             </gradientstop>
             <gradientstop position="1.000000000000000">
              <color alpha="255">
               <red>0</red>
               <green>128</green>
               <blue>0</blue>
              </color>
             </gradientstop>
            </gradient>
           </brush>
          </colorrole>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>243</red>
             <green>243</green>
             <blue>243</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base">
           <brush brushstyle="LinearGradientPattern">
            <gradient startx="0.000000000000000" starty="0.000000000000000" endx="0.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
             <gradientstop position="0.000000000000000">
              <color alpha="255">
               <red>144</red>
               <green>238</green>
               <blue>144</blue>
              </color>
             </gradientstop>
             <gradientstop position="1.000000000000000">
              <color alpha="255">
               <red>0</red>
               <green>128</green>
               <blue>0</blue>
              </color>
             </gradientstop>
            </gradient>
           </brush>
          </colorrole>
          <colorrole role="Window">
           <brush brushstyle="LinearGradientPattern">
            <gradient startx="0.000000000000000" starty="0.000000000000000" endx="0.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
             <gradientstop position="0.000000000000000">
              <color alpha="255">
               <red>144</red>
               <green>238</green>
               <blue>144</blue>
              </color>
             </gradientstop>
             <gradientstop position="1.000000000000000">
              <color alpha="255">
               <red>0</red>
               <green>128</green>
               <blue>0</blue>
              </color>
             </gradientstop>
            </gradient>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="Button">
           <brush brushstyle="LinearGradientPattern">
            <gradient startx="0.000000000000000" starty="0.000000000000000" endx="0.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
             <gradientstop position="0.000000000000000">
              <color alpha="255">
               <red>144</red>
               <green>238</green>
               <blue>144</blue>
              </color>
             </gradientstop>
             <gradientstop position="1.000000000000000">
              <color alpha="255">
               <red>0</red>
               <green>128</green>
               <blue>0</blue>
              </color>
             </gradientstop>
            </gradient>
           </brush>
          </colorrole>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>120</red>
             <green>120</green>
             <blue>120</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base">
           <brush brushstyle="LinearGradientPattern">
            <gradient startx="0.000000000000000" starty="0.000000000000000" endx="0.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
             <gradientstop position="0.000000000000000">
              <color alpha="255">
               <red>144</red>
               <green>238</green>
               <blue>144</blue>
              </color>
             </gradientstop>
             <gradientstop position="1.000000000000000">
              <color alpha="255">
               <red>0</red>
               <green>128</green>
               <blue>0</blue>
              </color>
             </gradientstop>
            </gradient>
           </brush>
          </colorrole>
          <colorrole role="Window">
           <brush brushstyle="LinearGradientPattern">
            <gradient startx="0.000000000000000" starty="0.000000000000000" endx="0.000000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
             <gradientstop position="0.000000000000000">
              <color alpha="255">
               <red>144</red>
               <green>238</green>
               <blue>144</blue>
              </color>
             </gradientstop>
             <gradientstop position="1.000000000000000">
              <color alpha="255">
               <red>0</red>
               <green>128</green>
               <blue>0</blue>
              </color>
             </gradientstop>
            </gradient>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton 
{
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 lightgreen, stop: 1 green);
    border-color: #008000;
    border-style: inset;
    border-width: 3px;
    border-radius: 10px;
}

QPushButton:hover 
{
	  border-style: outset;
}

QPushButton:flat 
{
    border: none; 
}

QPushButton:default 
{
    border-color: #008000;
}

QPushButton:pressed 
{
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 royalblue, stop: 1 dodgerblue);
    border-style: inset;
}</string>
       </property>
       <property name="text">
        <string>Принять</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="SoundAround.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>accept_btn</sender>
   <signal>clicked()</signal>
   <receiver>ConfigsForm</receiver>
   <slot>acceptSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>221</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>292</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialView_checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>ConfigsForm</receiver>
   <slot>changeDialViewSlot(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>80</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderView_checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>ConfigsForm</receiver>
   <slot>changeSliderViewSlot(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>changeDialViewSlot(bool)</slot>
  <slot>changeSliderViewSlot(bool)</slot>
  <slot>acceptSlot()</slot>
 </slots>
</ui>
